modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -544,6 +544,7 @@
     <string name="podcast_no_season">No season</string>
     <string name="podcast_season_x">Season %d</string>
     <string name="podcast_refresh_artwork">Refresh artwork</string>
+    <string name="podcast_no_ratings">No ratings</string>
```

Comment:
Based on this comment (pdeCcb-3ic-p2#comment-2726), we might want to update the title to "Not enough ratings".

Reply:
👍 Updated in https://github.com/Automattic/pocket-casts-android/pull/1453/commits/2017ec11bc20a8adf27d799cf9b7a84edf7211d4